### PR TITLE
Transfer Documentation to unsafePerformSync

### PR DESCRIPTION
**Pull request by ChristopherDavenport**
Updates tut Documentation for Scalaz 7.2. 

Partial coverage of #829

### CODE REVIEW

File: docs/src/main/tut/json.md [hunk 1]
```diff
@@ -177,8 +177,8 @@ val http4s = repos("http4s")
 
 val stargazers = http4s.map(_.map(_.stargazers_count).mkString("\n"))
 
-// Run has been separated into its own line for tut to compile without hanging. 
-stargazers.run
+// UnsafePerformSync has been separated into its own line for tut to compile without hanging.
```
Comment (reactormonk) [inline docs/src/main/tut/json.md]: ... could you make this lowercase? I think in this case it trumps the "sentence starts with capital letter".

Comment (ChristopherDavenport) [inline docs/src/main/tut/json.md]: Very easy. 2 seconds.

**Comment by rossabaker** [comment]
Hmm, do we want to go ahead and do this for the v0.15 docs?  I imagine it will cherry-pick cleanly.

**Comment by reactormonk** [comment]
@rossabaker Not cleanly, and it would also require a switch to 7.2 default build, because otherwise `docs/tut` will fail without additional settings.

**Pull request merged**